### ISSUE TITLE
fixed version doesn't work either

### ISSUE THREAD

**Issue by kenqkio** [zoony1337/TikTok-Shares-Botter#36]

**Comment by nicks-codes** [comment]
ya no shares go through

**Comment by crazymoose77756** [comment]
is there a fix for this yet

**Comment by r-ex** [comment]
> 
> 
> is there a fix for this yet

Not possible without significant change. TikTok now requires a sessionid in cookies when shares are counted, meaning that an account is needed.

**Comment by crazymoose77756** [comment]
so would the solution be just signing in with an account or something like that?

**Comment by kenqkio** [comment]
> 
> 
> so would the solution be just signing in with an account or something like that?

i think that there is a limit of share for an account so i think that there will never be a patch...

**Comment by crazymoose77756** [comment]
hopefully it will work again

**Comment by kenqkio** [comment]
> 
> 
> hopefully it will work again

🤞

**Comment by r-ex** [comment]
One solution, although lengthier than most would be willing to do, would be to use the session/user cookies that are present when sharing it usually for the request, although it doesn't seem that this would be efficient or possible in some scenarios.

**Comment by kenqkio** [comment]
> 
> 
> One solution, although lengthier than most would be willing to do, would be to use the session/user cookies that are present when sharing it usually for the request, although it doesn't seem that this would be efficient or possible in some scenarios.

but there is the limit per account

**Comment by r-ex** [comment]
> 
> 
> > One solution, although lengthier than most would be willing to do, would be to use the session/user cookies that are present when sharing it usually for the request, although it doesn't seem that this would be efficient or possible in some scenarios.
> 
> but there is the limit per account

correct, hence why i said "it doesn't seem that this would be efficient **or possible in some scenarios**"

**Comment by r-ex** [comment]
Also please note that the creator of this repo does not care for the longevity of the project, it wasn't intended to be used on the platform, more as a statement of how easy it was to abuse.

**Comment by Nyumat** [comment]
@r-ex I agree, the bot is most likely permanently patched unless someone can find a way to make a new session on every iteration of the program, but Tik tok countered this with making Oauth mandatory, so guess we have to cut the losses.

**Comment by kenqkio** [comment]
> 
> 
> Also please note that the creator of this repo does not care for the longevity of the project, it wasn't intended to be used on the platform, more as a statement of how easy it was to abuse.

yeah sure, its kinda useless, i just wanted to try if a lot of shares would trigger my followers

**Comment by r-ex** [comment]
> 
> 
> > Also please note that the creator of this repo does not care for the longevity of the project, it wasn't intended to be used on the platform, more as a statement of how easy it was to abuse.
> 
> yeah sure, its kinda useless, i just wanted to try if a lot of shares would trigger my followers

Shares don't massively effect reach, good content does.



> 
> 
> @r-ex I agree, the bot is most likely permanently patched unless someone can find a way to make a new session on every iteration of the program, but Tik tok countered this with making Oauth mandatory, so guess we have to cut the losses.

Yeah, sadly. It was never going to last as a tool, however, due to the nature of code being shared on github means that they can simply look at the source code and figure a way to fix it really fast, as they clearly already have.

**Comment by Nyumat** [comment]
This repo was on the front page so I'm willing to bet that ByteDance(TikTok parent company) developers saw it and were like "Let's end this." I have a ton of respect for the creator of this tool though, the documentation for the Tik Tok API was kinda weak. Maybe someday in the future i'll try and make an automated selenium bot, but we'll see how this all plays out.

**Comment by kenqkio** [comment]
> 
> 
> This repo was on the front page so I'm willing to bet that ByteDance(TikTok parent company) developers saw it and were like "Let's end this." I have a ton of respect for the creator of this tool though, the documentation for the Tik Tok API was kinda weak. Maybe someday in the future i'll try and make an automated selenium bot, but we'll see how this all plays out.

so do you mean that one day maybe a new tool should work ?

**Comment by r-ex** [comment]
> 
> 
> > This repo was on the front page so I'm willing to bet that ByteDance(TikTok parent company) developers saw it and were like "Let's end this." I have a ton of respect for the creator of this tool though, the documentation for the Tik Tok API was kinda weak. Maybe someday in the future i'll try and make an automated selenium bot, but we'll see how this all plays out.
> 
> so do you mean that one day maybe a new tool should work ?

I wouldn't count on it, and even if it did, it wouldn't be as effective as this once was.
